packages/mui-base/src/Select/Select.test.tsx

```diff
@@ -20,7 +20,7 @@ describe('<Select />', () => {
   const { render } = createRenderer();
 
   const componentToTest = (
-    <Select defaultListboxOpen slotProps={{ popper: { disablePortal: true } }}>
+    <Select defaultListboxOpen defaultValue={1} slotProps={{ popper: { disablePortal: true } }}>
```

Comment:
This is so that the ZWS is not rendered, or else `className="notranslate"` will cause the "test disabling class generation" part of `describeConformanceUnstyled` to fail